addons/survey/static/src/scss/survey_templates_form.scss

```diff
@@ -432,17 +432,34 @@ _::-webkit-full-page-media, _:future, :root .o_survey_wrap {
         color: white; // not bootstrap to customize for survey_print only
         font-weight: bold; // not bootstrap to customize for survey_print only
 
-        &.bg-success, &.bg-danger {
-            opacity: 0.6;
-        }
         &.o_survey_selected {
             background-color: $gray-600;
             opacity: 1;
         }
-        i.fa-square-o, i.fa-circle-thin {
+        i.fa-circle-thin {
             display: none;
         }
     }
+
+    // default border looks ugly for the scored questions because container div also have border in
+    // print page to indicate correct / incorrect answers, so we remove it and add needed rules
+    fieldset[disabled] .o_survey_questions {
```

Comment:
I think the style you are overriding is defined specifically for "disabled" question types here:

![image](https://user-images.githubusercontent.com/41282536/161941672-6060909f-976c-47c0-88aa-222b6e0add7f.png)

Why not remove that specific style instead of adding rules on top of it?

Reply:
Hello @awa-odoo , 

Here we have to apply changes only for the result page and if we remove this style directly then it will affect the preview of the form when the user is filling it. So, due to that reason, we have to add rules on top of it.

Reply:
I think the `&:disabled` part of my image above is specifically for the "result page".
Can you double-check?

Reply:
It is not only specific to the result page. It is also for o_survey_form, o_survey_session_manage, o_survey_quick_access with o_survey_print.

Reply:
I'm not talking about where the css rule applies, I'm talking about the fact that we only display disabled fieldsets in the results page from what I remember.

Reply:
True we use this property for the result page only but we cannot modify that as we will need that property when the survey is no_scoring type and also we have one more class there o_survey_question_char_box where we will need our new property only. So, I am not changing anything currently.

Reply:
"o_survey_question_char_box" does not seem used in the actual code so I would simply remove it from the css.

I checked out your branch and did this change:

![image](https://user-images.githubusercontent.com/41282536/162757101-a8587ca6-1df6-43ea-8f2a-09117ae059bf.png)

And removed your block.

I think it's better to override the color/border/font-weight only if we are adding a background behind it.
It also allows to get rid of the introduced "o_survey_questions" which was a bit confusing.

What do you think about it?

Reply:
Ah indeed, then you can keep that class, but the rest of my comment should still apply.

Reply:
Done!